web/packages/design/src/DataTable/Pager/ClientsidePager/ClientsidePager.tsx

```diff
@@ -1,17 +1,36 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 import React from 'react';
 
-import { Flex, Text } from 'design';
+import { PageIndicatorText } from 'shared/components/Search';
+
+import { Flex } from 'design';
 import { CircleArrowLeft, CircleArrowRight } from 'design/Icon';
 
-import { StyledArrowBtn, StyledFetchMoreBtn } from './StyledPager';
-import usePager, { State, Props } from './usePager';
+import { StyledArrowBtn, StyledFetchMoreBtn } from '../StyledPager';
+
+import useClientsidePager, { State, Props } from './useClientsidePager';
 
 export default function Container(props: Props) {
-  const state = usePager(props);
-  return <Pager {...state} />;
+  const state = useClientsidePager(props);
+  return <ClientsidePager {...state} />;
 }
 
-export function Pager({
+export function ClientsidePager({
```

Comment:
```suggestion
export function ClientSidePager({
```

I find this a lot easier to read.